.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
```

Comment:
We can use Github's release UI since it gives us a chance to format the PR descriptions into a release notes.

```elixir
on:
  release:
    types: [published]
```

Reply:
Updated.